.github/workflows/native-build-development.yml

```diff
@@ -86,13 +86,12 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_API_TOKEN }}
           STATUS: ${{ job.status }}
         run: |
+          echo "The report step got status: ${STATUS}"
           sudo apt-get update -o Dir::Etc::sourcelist="sources.list" \
             -o Dir::Etc::sourceparts="-" -o APT::Get::List-Cleanup="0"
           sudo apt-get install -y gnupg2 gnupg-agent
           echo Installing SDKMAN
           curl -s "https://get.sdkman.io" | bash
           source ~/.sdkman/bin/sdkman-init.sh && \
-          sdk install kotlin 1.3.61 && \
-          sdk install kscript 2.9.0
-          echo "The report step got status: ${STATUS}"
-          kscript .github/NativeBuildReport.kts ${GITHUB_TOKEN} ${STATUS}
+          sdk install jbang 0.36.1
```

Comment:
There is a `jbangdev/jbang-action@v0.36.1` Github action which could be used here instead
eg.
```yaml
      - name: Index Catalog
        uses: jbangdev/jbang-action@v0.36.1
        with:
          script: indexcatalog@quarkusio
          args: . ./app/registry.json
          trust: https://github.com/quarkusio/jbang-catalog
```